lib/chef/knife/bmcs_helper.rb

```diff
@@ -93,6 +93,20 @@ def display_list(response, columns)
 
         warn_if_page_is_truncated(response)
       end
+
+      # Return a true or false with the confirmation result.
+      # Note: user prompt is bypassed with --yes to confirm automatically.
+      def confirm(prompt)
```

Comment:
There's a ui.confirm operation, https://docs.chef.io/plugin_knife_custom.html - we should use that.